src/main/interface/msp.c

```diff
@@ -1839,6 +1841,7 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
     case MSP_SET_BEEPER_CONFIG:
         beeperOffClearAll();
         setBeeperOffMask(sbufReadU32(src));
+        beeperConfigMutable()->dshotBeaconTone = sbufReadU8(src);
```

Comment:
This needs to be guarded by `if (sbufBytesRemaining(src) >= 1)` to prevent it from breaking with older apps.

Reply:
Argh, offcource. Thanks.